dashboard/app/views/levels/_dialogHelper.js

```diff
@@ -35,14 +35,24 @@
     // Use our prefabricated dialog content.
     $(".modal-body").append($("#" + type + "-dialogcontent").clone(true));
 
-    // Clicking the okay button in the dialog box dismisses it.
+    // Clicking the okay button in the dialog box dismisses it, and calls the callback.
     $(".modal-body #ok-button").click(function () {
       dialog.hide();
+      callback && callback();
+    });
+
+    // Clicking the cancel button in the dialog box dismisses it.
+    $(".modal-body #cancel-button").click(function () {
+      dialog.hide();
     });
 
     dialog.show();
   }
 
+  window.showStartOverDialog = function(callback) {
```

Comment:
One option would be to do what we do in App Lab -- have the button be the
standard color for that tool (gray/white in the case of pixelation), and
then just do red on hover.

On Mon, Aug 24, 2015 at 11:18 AM, davidsbailey <notifications@github.com>
wrote:

> In dashboard/app/views/levels/_dialogHelper.js
> <https://github.com/code-dot-org/code-dot-org/pull/3517#discussion_r37783480>
> :
>
> >      });
> >
> >      dialog.show();
> >    }
> >
> > +  window.showStartOverDialog = function(callback) {
>
> @sfilman <https://github.com/sfilman> do you have a vision for resolving
> the issue which Laurel mentioned (red button attracts rather than deters
> clicks)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/3517/files#r37783480>.
>